### PR TITLE
Strip workflow run mako

### PR DESCRIPTION
**Pull request by guerler**
This PR removes the workflow run mako and the html builders for basic parameters from the backend.

### CODE REVIEW

File: lib/galaxy/tools/parameters/basic.py [hunk 1]
```diff
@@ -21,25 +20,24 @@
...
 import galaxy.model
 from galaxy.util.bunch import Bunch
 
-log = logging.getLogger(__name__)
+log = logging.getLogger( __name__ )
 
 workflow_building_modes = Bunch( DISABLED=False, ENABLED=True, USE_HISTORY=1 )
 
-WORKFLOW_PARAMETER_REGULAR_EXPRESSION = re.compile( '''\$\{.+?\}''' )
+workflow_parameter_regular_expression = re.compile( '''\$\{.+?\}''' )
```
Comment (jmchilton) [inline lib/galaxy/tools/parameters/basic.py]: I'd switch this back - the PEP-8 standard is pretty clear and we don't have any stated modifications to it with respect to constant. https://www.python.org/dev/peps/pep-0008/#constants

Comment (guerler) [reply lib/galaxy/tools/parameters/basic.py]: Ok. Thx for looking into it.

**Comment by guerler** [comment]
@galaxybot test this

**Comment by guerler** [comment]
@galaxybot test this

**Comment by guerler** [comment]
@galaxybot test this.

**Comment by guerler** [comment]
@jmchilton thanks a lot for reviewing this.

**Pull request merged**